src/game.cpp

```diff
@@ -3530,14 +3530,19 @@ void game::load(std::string worldname, std::string name)
     draw();
 }
 
-void game::load_world_modfiles(WORLDPTR world)
+void game::load_world_modfiles( WORLDPTR world, bool silent )
 {
-    popup_nowait(_("Please wait while the world data loads...\nLoading core JSON..."));
```

Comment:
Alternatively the implementation of `popup_nowait` could be changed to a no-op when running unit tests. This wouldn't be brittle to any further usage of the function. One option could be to reword the `debug_fatal` global into a more generic indicator of running unit tests.